app/views/announcements/_show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1><%= announcement.id %></h1>
```

Comment:
Wait, why would you want to show an announcement?

Reply:
I was thinking, if the announcement is too long to be read comfortably from the side panel, you can click and enlarge it...? Should I do away with it?

Reply:
Hmm how about a "Show more..." button after the (truncated) text?